core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -831,11 +847,70 @@ public void outboundWireSize(long bytes) {
   static final class ChannelBufferMeter {
     private final AtomicLong bufferUsed = new AtomicLong();
 
-    public long addAndGet(long newBytesUsed) {
+    @VisibleForTesting
+    long addAndGet(long newBytesUsed) {
       return bufferUsed.addAndGet(newBytesUsed);
     }
   }
 
+  /**
+   * Used for retry throttling.
+   */
+  static final class Throttle {
+    final int maxTokens;
```

Comment:
Optional: maybe helpful to include comments (text from the retry design) saying what these values represent

Reply:
Done.